src/index.html

```diff
@@ -1,13 +1,438 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+    <title>CRAZYBABY</title>
+    <link rel="icon" href="images/icons/favicon.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
     <script type="text/javascript" src="scripts/main.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css">
+    <script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js"></script>
+
+    <link rel="stylesheet" href="styles/main.scss">
+  </head>
+
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__wrapper">
+          <div class="header__content">
+            <div class="top-actions" >
+              <div>
+                <a
+                  href="#"
+                  class="logo nav__item"
+                  >
+                  <img
+                  src="./images/logo.svg"
+                  alt="CRAZYBABY"
+                  class="header__logo"
+                  id="logo">
+                </a>
+              </div>
+              <div class="burger" id="burger"></div>
+
+              <nav class="navigation navigation--tablet-desktop">
+                <a
+                  href="#"
+                  class="navigation__item"
+                  >specs
+                </a>
+                <div class="navigation__item">
+                  <a
+                    href="#"
+                    class="navigation__item--language-en"
+                    >EN
+                  </a>
+                  <span>/</span>
+                  <a
+                    href="#"
+                    class="navigation__item--language-ua"
+                    >UA
+                  </a>
+                </div>
+                <a
+                  href="#"
+                  class="navigation__item button button__buy"
+                  >buy
+                </a>
+              </nav>
+            </div>
+          </div>
+
+          <div class="banner">
+            <div class="grid">
+              <div
+                class="banner__photo
+                    grid__item grid__item--desktop-5-11
+                    grid__item--tablet-5-8">
+                <div class="banner__photo">
+                </div>
+              </div>
+              <div
+                class="banner__info
+                      grid__item
+                      grid__item--desktop-1-4
+                      grid__item--tablet-1-4 ">
```

Comment:
The same here, all over the code.
```suggestion
              <div
                class="banner__info
                      grid__item
                      grid__item--desktop-1-4
                      grid__item--tablet-1-4 "
               >
```